apps/trading-e2e/src/integration/withdrawals.feature

```diff
@@ -0,0 +1,67 @@
+Feature: Withdrawals
+
+  Scenario: Can prepare a withdrawal
+    Given I navigate to withdrawals page
+    And I connect eth wallet
+    And I connect vega wallet
+    And I can see the withdrawals warning info
+    And I can see the from ethereum address field is automatically populated with current connected key
+    And I select asset 'tBtc'
+    And I input '100' tokens to withdraw
+    And the withdraw button is enabled
+    When I click the withdraw button
+    Then I am redirected to the finalise withdrawals page
+
+  Scenario: Can enter a different withdrawal address
+    Given I navigate to withdrawals page
+    And I can see an eth address is already filled in the withdraw to field
+    When I click the enter manually button
+    Then I can enter a new eth address
+
+  Scenario: Form field validation if fields are incomplete
+    Given I navigate to withdrawals page
+    When I click the withdraw button
+    Then I can see validation errors on incomplete fields
+
+  Scenario: Can prepare a withdrawal to send to another eth wallet
```

Comment:
I think this is a dupe of "Can enter a different withdrawal address"

Reply:
update as part of     feat/104-withdraws